segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorage.java

```diff
@@ -178,7 +174,9 @@ public ChunkedSegmentStorage(int containerId, ChunkStorage chunkStorage, ChunkMe
 
         // Now bootstrap
         log.debug("{} STORAGE BOOT: Started.", logPrefix);
-        return this.systemJournal.bootstrap(epoch).thenRunAsync(() -> log.debug("{} STORAGE BOOT: Ended.", logPrefix), executor);
+        return this.systemJournal.bootstrap(epoch)
+                .thenRunAsync(() -> garbageCollector.initialize())
+                .thenRunAsync(() -> log.debug("{} STORAGE BOOT: Ended.", logPrefix), executor);
```

Comment:
Why async on this tiny one? If you don't have debug enabled, this won't even do anything?

Please try to reduce the number of `Async` calls, especially if the code you execute is very simple.

Reply:
Fixed